### PR TITLE
[hi] Localize cloud-provider.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/cloud-provider.md` to Hindi `content/hi/docs/reference/glossary/cloud-provider.md`

Fixes :- #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  - community
+---
+
+एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता है।
+
+<!--more-->
+
+क्लाउड प्रदाता, जिन्हें कभी-कभी क्लाउड सेवा प्रदाता (सीएसपी) कहा जाता है, क्लाउड कंप्यूटिंग प्लेटफॉर्म या सेवाएं प्रदान करते हैं।
+
+कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जब आप उसके ऊपरों लेयर्स का प्रबंधन करते हैं जैसे कुबेरनेट्स क्लस्टर चलाना।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/cloud-provider.md:20]: This makes the sentence a little better to read. Thoughts WDYT @bishal7679? 
```suggestion
कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर, स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जबकि आप उसके ऊपरी लेयर्स का प्रबंधन करते हैं जैसे कि  कुबेरनेट्स क्लस्टर चलाना।
```

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कण्ट्रोल प्लेन के साथ-साथ {{< glossary_tooltip term_id="नोड" text="nodes" >}} और यह अवसंरचना जिस पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य लोड बैलेंसर्स जैसे तत्व के लिए जिम्मेदार है।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/cloud-provider.md:22]: This makes the sentence a little better to read. Thoughts WDYT @bishal7679?
```suggestion
आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कण्ट्रोल प्लेन के साथ-साथ {{< glossary_tooltip term_id="नोड" text="nodes" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```

**Review by Babapool — COMMENTED** [review]
Hope you find these suggestions useful:

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान प्रदान करते हैं।
+
+<!--more-->
+
+क्लाउड प्रदाता, जिन्हें कभी-कभी क्लाउड सेवा प्रदाता (CSP) भी कहा जाता है, क्लाउड कंप्यूटिंग प्लेटफॉर्म या सेवाएं प्रदान करते हैं।
+
+कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर, स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जबकि आप उसके ऊपरी लेयर्स का प्रबंधन करते हैं जैसे कि  कुबेरनेट्स क्लस्टर चलाना।
+
+आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कण्ट्रोल प्लेन के साथ-साथ {{< glossary_tooltip term_id="नोड" text="nodes" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-provider.md:22]:
```suggestion
आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कण्ट्रोल प्लेन के साथ-साथ {{< glossary_tooltip text="नोड" term_id="nodes" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Please could you amend this? It is the reason why the build is failing.

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+short_description: >
+  एक संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता है।
+
+aka:
+  - Cloud Service Provider
+tags:
+  - community
+---
+
+एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान प्रदान करते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-provider.md:14]:
```suggestion
एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता हैं।
```

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+<!--more-->
+
+क्लाउड प्रदाता, जिन्हें कभी-कभी क्लाउड सेवा प्रदाता (CSP) भी कहा जाता है, क्लाउड कंप्यूटिंग प्लेटफॉर्म या सेवाएं प्रदान करते हैं।
+
+कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर, स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जबकि आप उसके ऊपरी लेयर्स का प्रबंधन करते हैं जैसे कि  कुबेरनेट्स क्लस्टर चलाना।
+
+आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कण्ट्रोल प्लेन के साथ-साथ {{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-provider.md:22]:
```suggestion
आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कंट्रोल प्लेन के साथ-साथ {{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+title: क्लाउड प्रदाता (Cloud Provider)
+id: cloud-provider
+date: 2018-04-12
+short_description: >
+  एक संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता है।
+
+aka:
+  - Cloud Service Provider
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-provider.md:9]:
```suggestion
  - क्लाउड सेवा प्रदाता (Cloud Service Provider)
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
Just one slight nit based on the files from other localization.

**Review by Babapool — COMMENTED** [review]
Thanks!
/lgtm

File: content/hi/docs/reference/glossary/cloud-provider.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+
+एक व्यवसाय या अन्य संगठन जो क्लाउड कंप्यूटिंग प्लेटफॉर्म प्रदान करता हैं।
+
+<!--more-->
+
+क्लाउड प्रदाता, जिन्हें कभी-कभी क्लाउड सेवा प्रदाता (CSP) भी कहा जाता है, क्लाउड कंप्यूटिंग प्लेटफॉर्म या सेवाएं प्रदान करते हैं।
+
+कई क्लाउड प्रदाता प्रबंधित अवसंरचना प्रदान करते हैं (जिन्हें Infrastructure as a Service या IaaS भी कहा जाता है)। प्रबंधित अवसंरचना के साथ क्लाउड प्रदाता सर्वर, स्टोरेज और नेटवर्किंग प्रदान करने के लिए जिम्मेदार है जबकि आप उसके ऊपरी लेयर्स का प्रबंधन करते हैं जैसे कि  कुबेरनेट्स क्लस्टर चलाना।
+
+आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कंट्रोल प्लेन के साथ-साथ {{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cloud-provider.md:22]: Too many "साथ" in one sentence. Eliminated this to retain conciseness without taking away from the meaning.
```suggestion
आप कुबेरनेट्स को एक प्रबंधित सेवा के रूप में भी पा सकते हैं; कई बार इसे Platform as a Service, या PaaS भी कहा जाता है। प्रबंधित कुबेरनेट्स के साथ, आपका क्लाउड प्रदाता कुबेरनेट्स कंट्रोल प्लेन{{< glossary_tooltip text="नोड" term_id="node" >}} और जिस अवसंरचना पर वे भरोसा करते हैं: नेटवर्किंग, स्टोरेज, और संभवतः अन्य तत्व जैसे लोड बैलेंसर्स के लिए जिम्मेदार है।
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I'd be happy to approve & lgtm once the changes are made.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**